### ISSUE TITLE
start 1st wda session in parallel

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/mcloud-ios#126]
we can't start wda itself on mac in parallel due to the Apple limitations but that's possible for wda sessions.
in current implementation start-wda and start-session do it consequently and as result old/problematic devices delay a lot farm stratup. 10 devices -> 10 extra minutes.

let review logic and made 1st session startup in parallel to speedup restarts

**Comment by vdelendik** [comment]
closing as won't fix because completely redesigned startup process and can do it in parallel for all iOS devices
